src/goto-programs/goto_inline_class.cpp

```diff
@@ -450,7 +450,7 @@ void goto_inlinet::insert_function_body(
   dest.destructive_insert(target, tmp);
 }
 
-void goto_inlinet::insert_function_nobody(
+void goto_inlinet::handle_function_nobody(
```

Comment:
Now would be a good time to add the doxygen header to this function as it isn't clear what handling a function without a body entails.